js/blocks/helpers/registerBlocks.js

```diff
@@ -9,13 +7,16 @@ import { registerBlockType } from '@wordpress/blocks';
  * Internal dependencies
  */
 import icons from '../../../assets/icons.json';
-import getBlockAttributes from './attributes';
-import { Edit } from '../components';
-import '../../../css/src/editor.scss';
```

Comment:
I'm not sure what this was for, it looks like this is enqueued

https://github.com/getblocklab/block-lab/blob/32e71bbbfbd89802a06ba363976d76e0862119fe/php/blocks/class-loader.php#L154-L159